docs/namespaced-broker-resources.md

```diff
@@ -0,0 +1,159 @@
+  uid: ae8e23ac-85d7-11e8-a4f3-2ae408f4a9e4
+spec:
+  description: basic plan
+  externalID: 4c6932e8-30ec-4af9-83d2-6e27286dbab3
+serviceBrokerName: example-ns-broker
+  serviceClassRef:
+    name: 25434f16-d762-41c7-bbdd-8045d7f74ca6
+status:
+  removedFromBrokerCatalog: false
+```
+
+The `ServiceInstance` resource has also been updated to allow you to use these resources just as you would the existing `ClusterServiceBroker`, `ClusterServiceClass` and `ClusterServicePlan` resources, except you will use them in the context of a namespace. For example, a `ServiceInstance` YAML that references a `ClusterServiceClass` and a `ClusterServicePlan` resource might look like this:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: example-mysql-instance
+  namespace: default
+spec:
+  clusterServiceClassExternalName: mysql-5-7
+  clusterServicePlanExternalName: basic
+```
+
+If you instead want to use the `ServiceClass` and `ServicePlan` namespace-scoped resources, the yaml might look like this:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: example-mysql-instance
+  namespace: default
+spec:
+  serviceClassExternalName: mysql-5-7
+  servicePlanExternalName: basic
+```
+
+In order to successfully create the resource using the namespace-scoped `ServiceClass` and `ServicePlan`, those resources must exist 
+in the same namespace as the `ServiceInstance` that you are creating. If a user attempts to create a `ServiceInstance` in a namespace 
+using a `ServiceClass` and `ServicePlan` that are contained in a different namespace, they request will fail.
+
+## Further Restricting Plan Access
+
+The use of namespace-scoped resources enables you to register brokers within a given namespace and leverage RBAC in order to control who can 
+provision services in that namespace. By default, all service classes and plans from that broker will be available to users of the namespace. When registering a broker, catalog restrictions can be specified in order to restrict what plans are available within a given namespace. This allows you to specify that in the `developer` namespace, only plans named `basic` can be created. The YAML to accomplish this might look like:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBroker
+metadata:
+  name: example-ns-broker
+  namespace: ns-broker
+spec:
+  authInfo:
+    basic:
+      secretRef:
+        name: my-service-broker-auth
+        namespace: broker
+  url: http://my-service-broker.broker.svc.cluster.local
+  catalogRestrictions:
+    servicePlan:
+    - "spec.externalName==basic"
+```
+
+When you combine the two capabilities, you can effectively restrict provisioning of service classes or plans to very specific namespaces. Production grade instances, for example, could be heavily restricted to a small subset of users. Other namespaces could be given access to other plans.
```

Comment:
when we add the doc for catalog restrictions it would be great to have a link here for more details on catalog restrictions.

Reply:
Yeah, I'm writing up catalog restrictions right now, so i'll link that when complete in a follow up PR.